Docs/sphinx_documentation/source/TimeStep.rst

```diff
@@ -88,3 +89,47 @@ The algorithm is further described in the following paper (and references therei
    http://www.sciencedirect.com/science/article/pii/S0021999198958909 :cite:`IAMR`
 
 
+Time Step -- BDS
+~~~~~~~~~~~~~~~~
+
+When we use the Bell-Dawson-Shubin (BDS) algorithm, we advance the solution in time using a
+three step procedure described below. In the notation,
+:math:`s` is a scalar field of the form :math:`s=s(x,y,z,t)`
+and :math:`{\bf u}=(u,v,w)` represents a known velocity field. :math:`s^n_{ijk}` represents
+the average value of :math:`s` over the cell with index :math:`(ijk)` at time :math:`t^n`.
+At each face the normal velocity (e.g., :math:`u_{i+1/2,j,k}`) is assumed constant
+over the time step.
+
+- **Step 1**: Construct a limited piecewise trilinear representation of the solution in
```

Comment:
Yes. How should I handle 2D vs 3D though? In this draft, I explained everything as if its was 3D. Should I add a separate 2D section? I could also add a citation for the 2D paper. 

Also, should I change **s** to phi ? They are equivalent right? I think this would make the explanation more coherent with the other algorithms.

More generally, how do you feel about the amount/form of the information?

Reply:
I think the level of information here is right; anything more detailed belongs in the Hydro docs or left to the paper. 

As far as 2D vs. 3D, I would just make some mention early on that for 2D, we replace trilinear with bilinear; in the following you give details for 3D. The 3D paper gives a nice summary of 2D, so I would say no need to reference the 2D paper explicitly. (I will push a change to the paper link though, to give access a a freely available copy on CCSE's website rather than the one you have to have SIAM access to see).

I don't see any need to change `s` to `phi`, but I don't think it would hurt either.

Reply:
Just say `trilinear (bilinear in 2D)` for each instance of trilinear.  The rest can stay as is - it's easy enough to back out the 2D case if the 3D is written out.

Reply:
...or just "linear" :)